app/routes/api/latest-contact-details.js

```diff
@@ -0,0 +1,32 @@
+import Joi from 'joi'
+import { getApplication } from '../../repositories/application-repository.js'
+
+export const latestContactDetailsHandlers = [
+  {
+    method: 'GET',
+    path: '/api/application/latest-contact-details/{ref}',
+    options: {
+      validate: {
+        params: Joi.object({
+          ref: Joi.string().valid()
+        })
+      },
+      handler: async (request, h) => {
+        const application = await getApplication(request.params.ref)
+        if (!application) {
```

Comment:
I think you will always get something back but might not have `dataValues` so:
```suggestion
        if (!application.dataValues) {
```

Reply:
Had a trial of this locally, and the way Ross had it originally seems right based on what comes back from the DB. result is completely null when ref not founnd